runtime/src/bank.rs

```diff
@@ -320,10 +323,10 @@ impl BankRc {
 pub struct LoadAndExecuteTransactionsOutput {
```

Comment:
are the comments in this struct still correct?

Reply:
Nope, fixed